cmd/corald/handlers/action.go

```diff
@@ -0,0 +1,92 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/ardanlabs/kit/cfg"
+	"github.com/ardanlabs/kit/web"
+	"github.com/coralproject/shelf/internal/sponge/item"
+	"github.com/coralproject/shelf/internal/talk"
+)
+
+// actionHandle maintains the set of handlers for the action api.
+type actionHandle struct{}
+
+// Action fronts the access to the action service functionality.
+var Action actionHandle
+
+var (
+	// ErrParameterNotFound is action not found error.
+	ErrParameterNotFound = errors.New("Parameter not found")
+)
+
+//==============================================================================
+
+// Create an action (flag, like, etc) on an Item.
+// 200 Success, 404 Not Found, 500 Internal
+func (actionHandle) Create(c *web.Context) error {
+
+	spongedURL := cfg.MustURL("SPONGED_URL").String()
+
+	// Get the user's item_id.
+	var user item.Item
```

Comment:
We are actually only passing the user_id and not the whole item. Maybe we should have this access point (POST & DELETE) as 

/v1/action/:action/on/item/:item_key/by/:user_key

instead of anything in the body